blog/_posts/2023-04-18-integrating-scalajs-with-vite.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: blog-detail
+post-type: blog
+by: Sébastien Doeraene
+title: Integrating Scala.js with Vite
+description: Using the new vite-plugin-scalajs to better integrate Scala.js with the Vite build tool.
+---
+
+One of the cornerstones of Scala.js is its ability to interoperate with JavaScript, in order to leverage its massive ecosystem.
+At the language level, we have good foundations and good tooling (namely [ScalablyTyped](https://scalablytyped.org/)) to reach out to JavaScript libraries.
+However, at the tooling level, Scala.js remains centered around Scala tooling.
+While this is good for the *coding* part of development, it is not so great for iterative feedback loops with the browser, or integration of other front-end assets.
+
+The JavaScript ecosystem features a lot of great tooling in that respect.
+[Vite](https://vitejs.dev/), in particular, is an interactive-first build tool for front-end development.
+It provides live-reloading with fast refresh cycles.
+Leveraging that tooling in the context of Scala.js would improve our ability to rapidly develop Scala.js applications.
+
+In this post, we introduce [`vite-plugin-scalajs`](https://github.com/scala-js/vite-plugin-scalajs), a Vite plugin to integrate a Scala.js codebase into the Vite development pipeline.
+In addition, we present a new tutorial series for Scala.js that integrates [Laminar](https://laminar.dev/) and ScalablyTyped with Vite.
+
```

Comment:
Screenshot? Short demo video? Here, and perhaps below, too? No one loves walls of text.